app/views/trn_submissions/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <div class="govuk-panel govuk-panel--confirmation trn-submission-success-panel">
+          <div class="govuk-grid-row">
+            <div class="trn-submission-success-panel__check">
+              &check;
+            </div>
+            <div class="trn-submission-success-panel__content">
+              <h1 class="govuk-panel__title">
+                Successfully submitted
+              </h1>
+              <div class="govuk-panel__body">
+                <span class="trn-submission-success-panel__emphasis">You have submitted the trainee for TRN registration</span> - we will update their status when we receive the TRN
+              </div>
+            </div>
+          </div>
+        </div>
+        <a class="govuk-link" href="/trainees">Back to trainee teacher list</a>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Most of this can be replaced with https://dfe-digital.github.io/govuk-components/#panel

Reply:
Good plan. Are you ok if I leave as is then as we'll replace it anyway?